src/commands/admin/activity.ts

```diff
@@ -1,41 +1,53 @@
-import { ActivityType, Message } from 'discord.js';
+import { ActivityType, ApplicationCommandOptionData, ApplicationCommandPermissionData, CommandInteraction, Message } from 'discord.js';
 import { BOT, DB } from '@root/config';
-import { botMasterPerms } from '@lib/permissions';
+import { BOTMASTER_PERMS } from '@lib/permissions';
 import { Command } from '@lib/types/Command';
 
+const args = ['PLAYING', 'STREAMING', 'LISTENING', 'WATCHING', 'COMPETING'];
+
 export default class extends Command {
 
-	description = `Sets ${BOT.NAME}'s activity to the given type and content`;
-	usage = '<type>|<content>';
+	description = `Sets ${BOT.NAME}'s activity to the given status and content`;
+	usage = '<status>|<content>';
+	tempPermissions: ApplicationCommandPermissionData[] = [BOTMASTER_PERMS];
+
+	options: ApplicationCommandOptionData[] = [
+		{
+			name: 'status',
+			description: 'The activity status (ex: Playing).',
```

Comment:
```suggestion
			description: 'The activity status.',
```
Example not needed due to the choice restrictions

Reply:
good shout